external/mqtt/src/test/scala/org/apache/spark/streaming/mqtt/MQTTStreamSuite.scala

```diff
@@ -113,7 +115,8 @@ class MQTTStreamSuite extends FunSuite with Eventually with BeforeAndAfter {
   }
 
   private def findFreePort(): Int = {
```

Comment:
We don't have a test utilities subproject, so this ends up getting duplicated, but note that we also have duplication of classes like LocalSparkContext; fixing this broader issue is outside the scope of this PR (there's a few JIRAs to track the creation of a "test utilities" project, though).